src/modal.js

```diff
@@ -0,0 +1,22 @@
+import React, {useState} from 'react';
+import ReactModal from 'react-modal';
+
+
+export function VidModal ({modalState, funcModalToFalse, HandleKeyDown}){
```

Comment:
This name: "funcModelToFalse" is kind of a hard to understand name. Can you change it to something that is a bit more clear? 

is it something like "onCancel?"

Reply:
Yeah, its job is to set the state of Modal being open to false. Will rework the name.

Reply:
I think its more of a handler for when the user is trying to close the modal. It wouldn't make sense for this specifically always close the modal, because then why not just include that state directly in the component? I think onCancel or onClose may be good?